action.yml

```diff
@@ -2,9 +2,16 @@
 name: 'Compose a GitHub Actions matrix'
 description: "Combine a base matrix with additional parameters"
 inputs:
-  base_matrix_url:
-    description: 'URL to the file containing the base matrix, encoded as JSON'
-    required: true
+  repository:
+    description: 'Repository to clone'
+    required: false
```

Comment:
Optional without default value? Wouldn't it crash later?

Reply:
In testing it appears to work as I expected. I think it has a value of `null` and then `actions/checkout` does the right thing.